keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -3,703 +3,931 @@
+## Motivation
 
-This KEP describes the above components, defines our goals and target use-cases, and sets the scope of the proposal by defining higher level objectives.  The vocabulary section defines terminology.  Relationships between the APIs are provided to illustrate the interconnections between object storage APIs, users' workloads, and object store service instances.  Lastly, the document describes the proposed API specs, the create and delete workflows, and the gRPC spec.
+File and block storage are treated as first class citizens in the Kubernetes ecosystem via CSI.  Workloads using CSI volumes enjoy the benefits of portability across vendors and across Kubernetes clusters without the need to change application manifests. **An equivalent standard does not exist for Object storage.**
 
-# Motivation
-
-File and block are first class citizens within the Kubernetes ecosystem.  Object, though very different under the hood, is a popular means of storing data, especially against very large data sources.   As such, we feel it is in the interest of the community to integrate object storage into Kubernetes, supported by the SIG-Storage community.  In doing so, we can provide Kubernetes cluster users and administrators a normalized and familiar means of managing object storage. 
-When a workload (app pod, deployment, configs) is moved to another cluster, as long as the new cluster runs a COSI controller that supports the same object protocol, then the workload can be moved to a new cluster without requiring any changes in the user manifests.
+Object storage has been rising in popularity in the recent years as an alternative form of storage to filesystems and block devices. Object storage paradigm promotes disaggregation of compute and storage. This is done by making data available over the network, rather than locally. Disaggregated architectures allow compute workloads to be stateless, which consequently makes them easier to manage, scale and automate.
 
 ## User Stories
 
-### Admin
+We define 3 kinds of stakeholders:
+
++ **Admins** - Members with escalated privileges and authority to manage site wide/cluster wide policies to control access, assign storage quotas and other concerns related to hardware or application resources.
++ **Application Developer/Operator (User)** - Members with privileges to run workloads, request storage for them, and manage resources inside a specific namespace.
++ **Object Storage Provider (OSP)** - Vendors offering Object Storage capabilities
+
+#### Admin
+
++ Establish cluster/site wide policies for data redundancy, durability and other data related parameters
++ Establish cluster/site wide policies for access control over buckets
++ Enable/restrict users who can create, delete and reuse buckets
```

Comment:
How admin exactly restricting the users, is it with help of `BucketClass` or `BucketAccessClass`?

Reply:
Neither, a new idea called referencePolicy is being proposed for this. Will add details during the next meeting